# Single-Cell Integration
Mathematical optimization methods in Single-Cell integration
# * non-negative Matrix Factorization* 
 | Label | Title  | Code | Model |Description | Others|
| ------------- | ------------- |------------- | ------------- | ------------- | ------------- |
|2016_Bioinformatics [paper](https://academic.oup.com/bioinformatics/article/32/1/1/1743821?login=false)  |A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data  |[code](https://github.com/yangzi4/iNMF)  | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/161.png)| ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/162.png)|
|2019_Cell [paper](https://www.sciencedirect.com/science/article/pii/S0092867419305045)  |Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity  |[code](https://github.com/welch-lab/liger) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/191.png)|Content Cell  |Content Cell |
|2023_TKDE [paper](https://ieeexplore.ieee.org/document/9893402)  |Structured Sparse Non-negative Matrix Factorization with L2,0-Norm for scRNA-seq Data Analysis  |[code](https://github.com/wenwenmin/SSNMF)  | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/231.png)| Content Cell  |Content Cell  |
|2022_Nature Communications [paper](https://www.nature.com/articles/s41467-022-28431-4)  |UINMF performs mosaic integration of single-cell multi-omic datasets using nonnegative matrix factorization  |[code](https://github.com/welch-lab/liger) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/221.png)  | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/222.png)  | Content Cell |
|2023_Nature Communications [paper](https://www.nature.com/articles/s41467-023-43019-2)  |Paired single-cell multi-omics data integration with Mowgli  |[code](http://github.com/cantinilab/mowgli) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/23NC1.png)  | Content Cell | Content Cell |
|2025_Nature Communications [paper](https://www.nature.com/articles/s41467-025-67088-7)  |CellMentor: cell-type aware dimensionality reduction for single-cell RNA-sequencing data |[code](https://github.com/petrenkokate/CellMentor) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/25NC1.png)  | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/25NC2.png)  | Content Cell |
|2025_Neural Comput & Applic [paper](https://link.springer.com/article/10.1007/s00521-025-11386-0)  |Integrative multi-omics approach for identifying and classifying high-risk patients in prostate cancer management  |[code] | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/25NCA1.png)  | Content Cell| Content Cell |
|2025_BMC Bioinformatics [paper](https://link.springer.com/article/10.1186/s12859-025-06301-2) |LONMF: a non-negative matrix factorization model based on graph Laplacian and optimal transmission for paired single-cell multi-omics data integration  |[code] | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/2025BMC1.png)  | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/25BMC2.png)| Content Cell |
|2025_Interdisciplinary Sciences: Computational Life Sciences [paper](https://link.springer.com/article/10.1007/s12539-025-00767-7) |Joint Low Rank Representation with Symmetric Orthogonal Decomposition for Clustering of scRNA-seq Data |[code] | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/24.png)  | | orthogonal |
|2024_BMC [paper](https://link.springer.com/article/10.1186/s12864-024-10729-w)|SPLHRNMTF: robust orthogonal non-negative matrix tri-factorization with self-paced learning and dual hypergraph regularization for predicting miRNA-disease associations|[code] | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/24BMC1.png)  | | orthogonal |
|2022_IEEE/ACM Trans Comput Biol Bioinform [paper](https://pubmed.ncbi.nlm.nih.gov/34546925/)|SOJNMF: Identifying Multidimensional Molecular Regulatory Modules by Sparse Orthogonality-Regularized Joint Non-Negative Matrix Factorization Algorithm|[code](https://github.com/JN-WYJ/SOJNMF) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/22TCBB.png)  | | orthogonal |
|2025_ BMC Genomics [paper](https://link.springer.com/article/10.1186/s12864-025-11462-8)|JSNMFuP: a unsupervised method for the integrative analysis of single-cell multi-omics data based on non-negative matrix factorization|[code](https://github.com/ZB-JN/JSNMFuP) | ![image](https://github.com/zhudafa/scRNA-seq-Data-Integration/blob/main/25BMCG1.png)  | | orthogonal |

